src/main/java/org/opensrp/web/rest/IdentifierSourceResource.java

```diff
@@ -0,0 +1,71 @@
+package org.opensrp.web.rest;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.opensrp.domain.IdentifierSource;
+import org.opensrp.service.IdentifierSourceService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequestMapping(value = "/rest/identifiersource")
+public class IdentifierSourceResource {
+
+	private static Logger logger = LoggerFactory.getLogger(IdentifierSourceResource.class.toString());
+
+	private IdentifierSourceService identifierSourceService;
+
+	protected ObjectMapper objectMapper;
+
+	public static final String IDENTIFIER = "identifier";
+
+	@Autowired
+	public void setObjectMapper(ObjectMapper objectMapper) {
```

Comment:
any specific reason to use autowiring in different places i.e. setter, constructor etc...

Reply:
Updated